aws-amp/adot/iam-read-amp-metrics.json

```
@@ -0,0 +1,15 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "aps:QueryMetrics",
+                "aps:GetSeries",
+                "aps:GetLabels",
+                "aps:GetMetricMetadata"
+            ],
+            "Resource": "*"
```

Suggest we parameterize the resource with the ARN for the AMP workspace.

```suggestion
            "Resource": "arn:aws:aps:::workspace/YOUR_AMP_WORKSPACE_ID"
```